internal/cli/plugin/install.go

```diff
@@ -31,16 +29,16 @@ type InstallOpts struct {
 	cli.GlobalOpts
 	cli.OutputOpts
 	Opts
-	AssetOpts
+	githubAssetManager *GithubAssetManager
```

Comment:
Could we consider dropping the `Manager` part?

I think this makes more sense:
```go
githubAsset *GithubAsset
```

Reply:
I agree, just changed it